modules/tensor_mechanics/doc/content/source/materials/ComputeLinearElasticPFFractureStress.md

```diff
@@ -214,16 +214,33 @@ This equation follows the standard Allen-Cahn and thus can be implemented in MOO
 Allen-Cahn kernels, TimeDerivative, AllenCahn, and ACInterface. There is now an action that automatically generates these kernels:
 NonconservedAction. See the +PhaseField module documentation+ for more information.
 
+## Pressure on the fracture surface
 
-## Example Input File Syntax
+As suggested by [!cite](CHUKWUDOZIE2019957), the work of pressure forces acting along each side of the cracks that is added to the total free energy can be approximated by
+\begin{equation}
+\int_{\Gamma}p(\mathbf{u}\cdot \mathbf{n})ds \approx -\int_{\Omega}p(\mathbf{u}\cdot\nabla d)
```

Comment:
I think our sign convention for normal is different from theirs. Do we really have a minus sign on RHS?

Reply:
you are correct!